ocs_ci/ocs/resources/pod.py

```
@@ -2596,20 +2596,27 @@ def verify_pods_upgraded(
     start_time = time.time()
     selector_label, selector_value = selector.split("=")
+    last_upgrade_error = None
+    verified_pods = set()
     while True:
         pod_count = 0
         pod_images = {}
         try:
             pods = get_all_pods(namespace, [selector_value], selector_label)
             pods_len = len(pods)
-            logger.info(f"Found {pods_len} pod(s) for selector: {selector}")
+            logger.debug(f"Found {pods_len} pod(s) for selector: {selector}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)